service/routes.py

```diff
@@ -157,6 +157,25 @@ def create_items(order_id):
 
     return make_response(jsonify(message), status.HTTP_201_CREATED)
 
+######################################################################
+# LIST ALL ITEMS FROM AN ORDER
+######################################################################
+@app.route("/orders/<int:order_id>/items", methods=["GET"])
+def list_items(order_id):
+    """List all of the Items from Order"""
+    app.logger.info("Request to list Items for Order with id: %s", order_id)
+    check_content_type("application/json")
+
+    # See if the order exists and abort if it doesn't
+    order = Order.find(order_id)
+    if not order:
+        abort(status.HTTP_404_NOT_FOUND, f"Order with id '{order_id}' was not found.")
+    # Create an item from the json data
+    items = OrderItem().all()
+    results = [item.serialize() for item in items]
```

Comment:
This will return all the items in the database, which may belong to different orders. In this API, we want all items of only one order specified by order_id. You can do something as follows (since you already have the order):

results = [item.serialize() for item in order.items]

Reply:
Thanks, done!